source/index.html.md

```diff
@@ -333,38 +333,53 @@ Example response (snippet):
     },
     "processed_at": "2018-07-09T12:58:56+00:00",
     "amount": 50000,
-    "currency": "EUR",
-    "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
+    "currency": "EUR"
 }
 ````
 
 
-## Recurring payments
+## Series of transactions
 
-Recurring payments enable you to repeatedly charge cardholders without having
-them to provide card information for subsequent payments.
+Subscriptions are supported using `series[]`. There are 3 types of
+subscriptions: `installment`, `recurring`, and `unscheduled`
+(merchant-initiated) credential on file.
 
-### Subscription concept
+A `recurring` series consist of transactions with the same amount being charged
+at regular intervals.
 
-Many PSPs have a subscription concept for supporting recurring payments.
-The first approved authorization is the initial recurring authorization (also
-known as "first in series"), all later authorizations are called subsequent
-authorizations. Clearhaus supports subscriptions in the form of recurring
-payments.
+An `installment` series is a special case of recurring where, additionally, the
+goods/services delivered by the merchant is delivered at the time of the
+first in series transactions, and the agreement ends at an agreed date or after
+an agreed number of withdrawals.
 
-### Repeatedly reserve money
+An `unscheduled` credential on file is a series of merchant-initiated
+transactions just like a `recurring` series, but with the exception that the
+intervals between withdrawals are irregular; instead, transactions are made
+ad-hoc.
 
-A recurring payment is made by making an authorization and setting `recurring`
-parameter to `true`. The first recurring payment for a given card could be made
-this way (notice that the amount may be zero):
+For subscriptions, the first in series transaction is cardholder-initiated
+transaction whereas all subsequent transactions er merchant-initiated.
+
+PSPs and merchants may store the cardholder's payment credentials to allow a
+returning cardholder to conveniently pay without entering the payment
+credentials again. (Explicit agreement with Clearhaus is required.)
+To store cardholder's payment credentials and use these stored payment
```

Comment:
*cardholders'

Reply:
For one auth you store "(a) cardholder's payment" credentials. Uhm, but then it should likely be "credential" (without the plural `s`).

What's better?
> To store cardholders' payment credentials (...)
> To store a cardholder's payment credential (...)

Reply:
An easy solution is to remove "cardholder's" from the sentence.